app/src/main/res/layout/activity_wallet_validation.xml

```diff
@@ -9,31 +9,50 @@
       android:id="@+id/wallet_validation_toolbar"
       layout="@layout/layout_app_bar"
       android:visibility="invisible"
```

Comment:
In order to avoid the "white bar" hiding the visible views while scrolling, in the wallet verification screen on the onboarding flow, the visibility here should be set to "gone".